js_basics_HT/task5_1.js

```diff
@@ -0,0 +1,9 @@
+//Зробів два віришення на вибір, друге мені більше до вподоби)
+
+//const radius = 15
+
+//console.log(Math.PI*Math.pow(radius,2))
+
+const radius = 15
+const S = (Math.PI*Math.pow(radius,2))
```

Comment:
```js
const circleArea = Math.PI * Math.pow(radius,2)
```

Reply:
remember about camelCase and meaningful names for variables

Reply:
S - means square (I'll replace it with Area)